javascript-timers/interval.js

```diff
@@ -0,0 +1,19 @@
+setInterval(countdown, 1000);
+var ticker = 1;
+function countdown() {
+  if (ticker === 1) {
+    document.querySelector('.four').textContent = '4';
+    ticker += 1;
+  } else if (ticker === 2) {
+    document.querySelector('.four').textContent = '3';
+    ticker += 1;
+  } else if (ticker === 3) {
+    document.querySelector('.four').textContent = '2';
+    ticker += 1;
+  } else if (ticker === 4) {
+    document.querySelector('.four').textContent = '1';
+    ticker += 1;
+  } else if (ticker === 5) {
+    document.querySelector('.four').textContent = '~~Earth Beloowwww Usssss~~';
+  }
```

Comment:
You need to `clearInterval` once everything has finished

Reply:
![1357437](https://user-images.githubusercontent.com/122063836/220804893-899a4cca-59ba-426e-842f-5e207f23c107.gif)